CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+### Added
+- Code ref parameter to switch on/off adding Playwright project names to code reference
```

Comment:
What is 'code ref parameter'? leave just 'parameter' or rename according to the correct naming.